orm/src/main/java/org/hibernate/search/hcore/impl/HibernateSearchSessionFactoryObserver.java

```diff
@@ -93,8 +110,40 @@ public void sessionFactoryCreated(SessionFactory factory) {
 		}
 	}
 
+	private void startTriggerAsyncBackendService(SessionFactory factory) {
```

Comment:
Is there a strong need for starting the async backend from within the ORM module? Could there not be a specific observer in the async backend module?